k8s.gcr.io/ensure-staging-repo.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+#
+# This script is used to create a new "staging" repo in GCR.  Each sub-project
+# that needs to publish container images should have their own staging repo.
+#
+# Each staging repo exists in its own GCP project, and is writable by a
+# dedicated googlegroup.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_PATH=$(readlink -f "$0")
+SCRIPT_DIR=$(dirname "$SCRIPT_PATH")
+source "${SCRIPT_DIR}/lib.sh"
```

Comment:
Isn't `source` a bashism? According to https://superuser.com/a/46146 the POSIX-y way is to just use `.` to source.

Reply:
interesting.